WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift

```diff
@@ -45,6 +45,11 @@ class LoginEmailViewController: LoginViewController, NUXKeyboardResponder {
         setupOnePasswordButtonIfNeeded()
         configureForWPComOnlyIfNeeded()
         addGoogleButton()
+
+        instructionLabel?.textAlignment = .left
+        errorLabel?.textAlignment = .right
```

Comment:
I'm not sure about this one in particular. It's a good reminder to take into account RTL layout when you need the text to display in the opposite alignment of the natural. But the bad part is that later on, we need to silence this warning manually adding that comment.

In the other hand, this scenario is quite rare, so it still might be good to keep this rule just in case.

Reply:
I feel the `.left` and `.right` errors should be the same. "Forcing text alignment to right is the inverse to natural language direction" sounds a bit condescending, as our designers realise this and have reasons for aligning opposite 😜

Reply:
@nheagy  That is the point for doing them separately:
- if `.left` you probably meant `.natural` and there's a lint for that.
- If `.right`, it's most probably a design decision, so we should make sure that in RTL layout, the alignment is `.left` instead of also `.right`.

Does it make sense? 😆
For sure the message is not the best one.